src/templates/template-common/utils.py

```diff
@@ -68,7 +85,7 @@ def setup_config(parser=None):
 
     config.backend = args.backend
 
-    return DictConfig(config)
+    return config
```

Comment:
Why we do not need `DictConfig` anymore ?

Reply:
Since we are using `config = OmegaConf.load(config_path)` so it already is a `DictConfig` data type

Reply:
So, it was missed in a previous PR